src/PyHyperScattering/PFEnergySeriesIntegrator.py

```diff
@@ -72,10 +73,12 @@ def integrateSingleImage(self,img):
             except IndexError:
                 en = float(img.energy)
             except AttributeError:
-                en = img.energy[0]
-                warnings.warn(f'Using the first energy value of {img.energy}, check that this is correct.',stacklevel=2)
+                try:
+                    en = img.energy[0]
+                    warnings.warn(f'Using the first energy value of {img.energy}, check that this is correct.',stacklevel=2)
```

Comment:
This warning may be problematic, if `img.energy` has multiple values, there is a risk of q shifting (see recent issue).
```suggestion
                    warnings.warn(f'Using the first energy value of {img.energy}, if this contains more than one energy, your data are likely wrong.',stacklevel=2)
```